Fauna/Response/QueryResponse.cs

```diff
@@ -1,3 +1,4 @@
+using System.Net;
 using Fauna.Mapping;
 using Fauna.Serialization;
 using System.Text.Json;
```

Comment:
Nit: Put all the `System.*` usings at the top.
```suggestion
using System.Net;
using System.Text.Json;
using Fauna.Mapping;
using Fauna.Serialization;
```